healthyregions/SDOHPlace
# Fix spatial res filter

This is a quick fix for the spatial resolution filter.

## How to Test

Run the app and click any of the spatial resolution filters. The filter action should be done
